### ISSUE TITLE
Page template is missing a `main` landmark

### ISSUE THREAD

**Issue by mtthwhggns** [apluslms/a-plus#595]
Specifying a `main` element would be important for accessibility. 

Reference: https://www.w3.org/TR/wai-aria-practices/examples/landmarks/main.html
Related: https://apluslms.github.io/accessibility-audit/patterns/fixable-a/semantic-markup-and-sequences/
